core/src/main/scala/chisel3/properties/Class.scala

```diff
@@ -89,10 +89,14 @@ class Class extends BaseModule {
 
   /** Internal state and logic to maintain a buffer of commands.
     */
-  private val _commands = new ArrayBuffer[Command]()
```

Comment:
By moving this to `Block`, the backing here is now a `VectorBuilder[Command]` instead of an `ArrayBuffer`.

Not sure if there's a preference here or more investigation/characterization of their usage needed, but just drawing attention in case it matters 👍 .